api/controllers/game/resolve-five.js

```diff
@@ -7,16 +7,16 @@ module.exports = async function (req, res) {
     const [game, player, cardToDiscard] = await Promise.all([promiseGame, promisePlayer, promiseCard]);
 
     if (!game.topCard) {
-      throw { message: 'game.snackbar.five.fiveDeckIsEmpty' };
+      throw { message: 'game.snackbar.oneOffs.five.fiveDeckIsEmpty' };
```

Comment:
Reorganized translation keys for one-off errors so they all live under `game.snackbar.oneOffs` with the general ones that are used in multiple places there, and move specific ones under their one-off rank e.g. `game.snackbar.oneOffs.five`